sites/headers-includes/header.html

```diff
@@ -1,16 +1,33 @@
 <header>
 	<div id="wb-bnr" class="container">
 		<div class="row">
-			{% if page.altLangPage %}{% include language/inc-languagetoggle.html %}{% endif %}
+			{% if page.altLangPage %}
+				{% include language/inc-languagetoggle.html %}
+			{% endif %}
 			{% include headers-includes/brand.html %}
-			{% unless page.nositesearch %}{% include headers-includes/sitesearch.html %}{% endunless %}
+			{% unless page.nositesearch %}
+				{% include headers-includes/sitesearch.html %}
+			{% endunless %}
 		</div>
 	</div>
-{%- unless page.nomenu -%}
-	{% include gcweb-menu/inc-sitemenu.html %}
-{%- endunless -%}
-{% include breadcrumbs/inc-breadcrumbs.html %}
-{%- if page.signing -%}
-	{% include headers-includes/sign-on-off.html %}
-{%- endif -%}
+	<hr>
+	<div class="container">
+		<div class="row">
+			{%- unless page.nomenu -%}
+				{%- if page.auth -%}
+					<div class="col-sm-9">
+				{%- else -%}
+					<div class="col-sm-12">
+				{%- endif -%}
+				{% include gcweb-menu/inc-sitemenu.html %}
+			{%- endunless -%}
+		</div>
+		{%- if page.auth -%}
+			<div class="col-sm-3">
```

Comment:
The parent element must be a <code>div.row</code>. Currently it is sibling which is not correct.